### PR TITLE
Leaderboard and MetricsCron

### PR DESCRIPTION
**Pull request by travisreed-wf**

### CODE REVIEW

File: templates/js/home.js [hunk 1]
```diff
@@ -6,9 +6,14 @@ $(document).ready(function() {
...
   game = new Game($('#game'));
   game.createInitialView();
+
+  $('#leaderboard').DataTable({
+    "order": [[1, "desc" ]]
+  });
 });
 
 
+
```
Comment (josephmcgovern-wf) [inline templates/js/home.js:12]: Lots of whitespace to remove

**Comment by josephmcgovern-wf** [comment]
@travisreed-wf could you go ahead and make the leaderboard it's own page?

**Comment by travisreed-wf** [comment]
yeah

**Comment by travisreed-wf** [comment]
@josephmcgovern-wf I went ahead and deployed this, moving the leaderboard to its own file and adding a new MetricCron

please let me know if you have any questions/concerns.

**Pull request closed without merging**